src/Form.svelte

```diff
@@ -3,11 +3,28 @@
     name: string;
     value: string | null;
     label?: string;
+    validate?: string;
     placeholder?: string;
     description: string;
     resolve?: boolean;
     editable: boolean;
+    error?: string;
   }
+
+  const validationTypes: { [index: string]: RegExp } = {
+    url: /^(https:|http:|www\.)\S+/,
```

Comment:
I wouldn't allow `www.` here as it's not technically valid and we don't want to have issues rendering these. Also make sure you add `\/\/` after the protocol.

Reply:
Perhaps also worth adding `ipfs://` for the future ;)

Reply:
Okay no problem, the `\/\/` is already taken care of with `\S` but we can be explicit and show it in the `(https:|http:)` group, like `(https:\/\/|http:\/\/)`.
Great idea with adding `ipfs://` :smiley:

Reply:
The issue with only having `\S` is that `http://` is considered valid :shrug:

Reply:
Hmm yeah, there are different regex rules I found that go into a lot of depth declaring all possible domains, but I think we should cover the basics.. Will build something more declarative

Reply:
It's a interesting topic very plenty of regex have been made, see eg https://mathiasbynens.be/demo/url-regex.

Reply:
Yeah it's better to be more permissive. Nothing worse than throwing an error on valid input.